CHANGES.rst

```diff
@@ -13,6 +13,9 @@ Deprecated
 Features added
 --------------
 
+* #12727: build: Allow --pdb to debug warnings when --fail-on-warning is specified.
```

Comment:
Could you use the option role for links? that would help users when they read the changelog. Thanks.

Reply:
Done.